### ISSUE TITLE
Android Only:  WebView reloads page on navigation 

### ISSUE THREAD

**Issue by cjohn001** [NativeScript/NativeScript#9515]
Hello together,
I am using a WebView to display a static webpage. On Android, I now have the problem, that when I transition away from the page (back button or forward to another page) the WebView reloads its content, leading to a short blinking which looks very suboptimal.  Is there a way I can disable this behavior until the bug is fixed? Not sure what event event might be responsible for it and how I can intercept it.

 - CLI: 8.0.2-next-08-13-2021-1128363962
 - Cross-platform modules: "@nativescript/core": "~8.1.0-sap.0",
 - Android Runtime: "@nativescript/android": "8.0.0",
 - XCode Version: 12.5.1

**Comment by farfromrefug** [comment]
@cjohn001 yes it is a normal behavior seeing how N is implemented. Should be fixed in 8.1.0.alpha1. Could you try? (look for the tag on npm)

**Comment by cjohn001** [comment]
Good Morning @farfromrefug , I had tested with 8.1.0.alpha1 as well, now repeated it. The problem still exists. Can you recommend a quick workaround till it is fixed?

**Comment by farfromrefug** [comment]
@cjohn001 ok i might need to reproduce it. it is really weird for it to still happen with 8.1.0. By any chance could you create a simple repro example?

**Comment by cjohn001** [comment]
I will try to do it tonight. Rather busy today.

**Comment by cjohn001** [comment]
@farfromrefug : Sorry for the late reply, but I could not find time earlier to build a demo showing the issue. Indeed the demo I provide, will show you two bugs.
First bug, referenced here, which causes flickering by the webview reloading the html page when back button is pressed.

Second bug: Appears on Android API Level 30 and above. Please note, since this month Google store requires target version 30 or above, hence I consider this bug as urgent as well. 
WebView cannot load from absolute file path.

[webview.zip](https://github.com/NativeScript/NativeScript/files/7025388/webview.zip)

**Comment by cjohn001** [comment]
@farfromrefug : Sorry to bother you with it, but as the issue is one of the last things that prevent me from an android release, I would like to ask you if the demo app I provided worked for you and if you are seeing the issues. Thanks for your feedback.

**Comment by farfromrefug** [comment]
@cjohn001 your demo app wont run here. Some error about `_domId`. Cant push it further (and dont know nothing about angular). But could you test with `@nativescript/core:8.1.0-alpha.7` and not the sap one?

**Comment by cjohn001** [comment]
@farfromrefug: I also tried with @nativescript/core:8.1.0-alpha.7  now. Still the same problem. 

>>  Some error about _domId
Not sure what the problem might be. It is a recent basic ng template I am using. And it should run with issues. What tns version have you installed? Could you please try to run with

ns run android --no-hmr

**Comment by farfromrefug** [comment]
@cjohn001 this is what i do and i am up to date on almost everything. It should work. Sorry i cant push further on that angular demo. I am still on holidays and wont have much time until mid september

**Comment by cjohn001** [comment]
@farfromrefug: You mentioned that the problem should have been fixed. Could you give me some directions. What you should considered the problem? I will than have a look at the core sources. Maybe I can solve it.

**Comment by cjohn001** [comment]
Update: I assume, the second bug my example shows (not the page reload on), i.e. loading local files, is related to the new way google requests file loading, see discussion here:

https://www.b4x.com/android/forum/threads/webview-access-denied-with-api-30.132424/

**Comment by farfromrefug** [comment]
@cjohn001 in 8.1.0 fragments are not destroyed on navigation forward which was causing a lot of issues. Now you can check if unloaded/loaded is still called on thw webview on navigation forward/back. That could be the issue in how webview handles those events

**Comment by cjohn001** [comment]
@farfromrefug, thanks for the directions. Unfortunately, I do not get it fixed. Seems my android understanding is insufficient. Would be great if you could have another look.
